bbb-common-web/src/main/java/org/bigbluebutton/api/model/request/JoinMeeting.java

```diff
@@ -52,8 +55,12 @@
 
     private String role;
 
+    @Valid
+    private Password joinPassword;

```

Comment:
```suggestion
    private JoinPassword joinPassword;

```

Should this be an instance of JoinPassword?

Reply:
It can be, but I prefer to follow the practice of "program to an interface not an implementation". 
e.g. You should do "List aList" instead of "ArrayList aList", since "List" is the interface and "ArrayList" is an implementation.
JoinPassword is an implementation of Password.